[debug] [2018-11-26T18:11:53.172Z] ----------------------------------------------------------------------
[debug] [2018-11-26T18:11:53.174Z] Command:       /usr/bin/node /usr/local/bin/firebase serve
[debug] [2018-11-26T18:11:53.174Z] CLI Version:   6.0.1
[debug] [2018-11-26T18:11:53.174Z] Platform:      linux
[debug] [2018-11-26T18:11:53.174Z] Node Version:  v8.11.4
[debug] [2018-11-26T18:11:53.175Z] Time:          Mon Nov 26 2018 19:11:53 GMT+0100 (CET)
[debug] [2018-11-26T18:11:53.175Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-11-26T18:11:53.183Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-11-26T18:11:53.183Z] > authorizing via signed-in user
[debug] [2018-11-26T18:11:53.183Z] [iam] checking project planning-poker-808fc for permissions ["firebase.projects.get"]
[debug] [2018-11-26T18:11:53.185Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/planning-poker-808fc:testIamPermissions  

[debug] [2018-11-26T18:11:54.233Z] <<< HTTP RESPONSE 200
[debug] [2018-11-26T18:11:54.236Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/planning-poker-808fc  
 
[debug] [2018-11-26T18:11:55.196Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/home/karol/Dokumenty/planning_poker_2.0'...
[info] 
[info] i  functions: Preparing to emulate functions.
[debug] [2018-11-26T18:11:55.801Z] Fetching environment
[debug] [2018-11-26T18:11:56.027Z] Error: EACCES: permission denied, open '/home/karol/.config/gcloud/application_default_credentials.json'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.writeFileSync (fs.js:1299:33)
    at module.exports (/usr/local/lib/node_modules/firebase-tools/lib/ensureDefaultCredentials.js:28:8)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:106:5)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:33)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:18:23
    at arrayMap (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9556:14)
    at _serve (/usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:16:26)
    at Command._action (/usr/local/lib/node_modules/firebase-tools/lib/commands/serve.js:58:12)
[error] 
[error] Error: An unexpected error has occurred.
